src/applications/caregivers/tests/unit/components/ApplicationDownloadLink.unit.spec.js

```diff
@@ -42,27 +46,23 @@ describe('CG <ApplicationDownloadLink>', () => {
 
   context('default behavior', () => {
     it('should render a download file button', () => {
-      const { container } = subject();
-      const link = $('va-link', container);
-      expect(link).to.exist;
-      expect(link).to.have.attr('text', content['button-download']);
+      const { selectors } = subject();
+      const { vaAlert, vaLink, vaLoadingIndicator } = selectors();
+      expect(vaLink).to.exist;
+      expect(vaAlert).to.not.exist;
+      expect(vaLoadingIndicator).to.not.exist;
+      expect(vaLink).to.have.attr('text', content['button-download']);
     });
   });
 
   context('when clicking the download file button', () => {
-    const triggerError = ({ container, status }) => {
-      const link = $('va-link', container);
```

Comment:
The default test checks that the appropriate elements exists. No need to do it in all the subsequent tests.